docs/labs/insecure-deserialization.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://best.openssf.org/assets/css/style.css">
+<link rel="stylesheet" href="checker.css">
+<script src="js-yaml.min.js"></script>
+<script src="checker.js"></script>
+<link rel="license" href="https://creativecommons.org/licenses/by/4.0/">
+
+<!-- See create_labs.md for how to create your own lab! -->
+
+<!-- Sample expected answer -->
+<script id="expected0" type="plain/text">
+  const data = JSON.parse(base64Decoded);
+</script>
+<!--
+-->
+<script id="expected1" type="plain/text">
+  if (data.username && typeof data.username == 'string' && data.username.length < 20) {
+</script>
+
+<!-- Full pattern of correct answer -->
+<script id="correct0" type="plain/text">
+ const\s+data = JSON\.parse\(base64Decoded\)\; \s*
+</script>
+<script id="correct1" type="plain/text">
+ if \(data\.username && typeof\s+data\.username == ('string'|"string"|`string`) && data\.username\.length < 20\) \{ \s*
```

Comment:
Again, in correct1 I think there should be spaces around `(` and `\.` and `\)` because whitespace is optional around them.

I think after typeof you should allow the following parameter to be surrounded by a parenthesized pair, that is, `typeof ( \( data \. username \) | data \. username )` - treating typeof as a function/method call isn't insane, and JavaScript would accept it.